x-pack/plugins/ingest_manager/public/applications/ingest_manager/sections/agent_config/list_page/components/create_config.tsx

```diff
@@ -70,11 +71,14 @@ export const CreateAgentConfigFlyout: React.FunctionComponent<Props> = ({
           />
         </h2>
       </EuiTitle>
+      <EuiSpacer size="m" />
       <EuiText size="s">
-        <FormattedMessage
-          id="xpack.ingestManager.createAgentConfig.flyoutTitleDescription"
-          defaultMessage="Agent configurations are used to manage settings across a group of agents. You can add integrations to your agent configuration to specify what data your agents collect. When you edit an agent configuration, you can use Fleet to deploy updates to a specified group of agents."
-        />
+        <p>
+          <FormattedMessage
+            id="xpack.ingestManager.createAgentConfig.flyoutTitleDescription"
+            defaultMessage="Agent configurations are used to manage settings across a group of agents. You can add integrations to your agent configuration to specify what data your agents collect. When you edit an agent configuration, you can use Fleet to deploy updates to a specified group of agents."
```

Comment:
This message needs work, but I want to drop play around with the build and try out a few variations to see how they read in the UI. I have https://github.com/elastic/kibana/issues/70023 open to review the text strings and make sure they follow EUI guidelines. I'll look at this message when I work on that issue.